os/fs/romfs/fs_romfs.c

```diff
@@ -656,6 +672,50 @@ static int romfs_dup(FAR const struct file *oldp, FAR struct file *newp)
 }
 
 /****************************************************************************
+ * Name: romfs_fstat
+ *
+ * Description:
+ *   Obtain information about an open file associated with the file
+ *   descriptor 'fd', and will write it to the area pointed to by 'buf'.
+ *
```

Comment:
input arguments and output is not explained in comments.

Reply:
Not sure what you mean - the sentence states that info from 'filep' is written into 'buf'. That clearly indicates input and output fields (const and no const do the same).
Besides there is no parameter description in each function in this file.